.travis.yml

```diff
@@ -92,11 +100,28 @@ install:
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then ulimit -S -n 2048; fi
-  - echo $MAIN_CMD $SETUP_CMD
-  - eval $MAIN_CMD $SETUP_CMD
+  - |
+    if [[ $NAME == 'minimal' ]] || [[ $NAME == 'full' ]] || [[ $NAME == 'Lint' ]] ||  [[ $NAME == 'osx' ]] ||  [[ $NAME == 'old numpy' ]] ||  [[ $NAME == 'numpy dev' ]]; then \
```

Comment:
In the longer term we could look at using build stages for things like this

https://docs.travis-ci.com/user/build-stages